base/services/w32time/W32Time.c

```diff
@@ -0,0 +1,370 @@
+/*
+ * PROJECT:     ReactOS W32Time Service
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Create W32Time Service that reqularly syncs clock to Internet Time
+ * COPYRIGHT:   Copyright 2018 Doug Lyons
+ */
+
+#include<windows.h>
+#include <debug.h>
+
+#include "timedate.h"
+
+SERVICE_STATUS ServiceStatus; 
+SERVICE_STATUS_HANDLE hStatus;
+static WCHAR ServiceName[] = L"W32Time";
+ 
+static VOID CALLBACK ServiceMain(int argc, char** argv); 
+void  ControlHandler(DWORD request); 
+int InitService();
```

Comment:
Empty parentheses mean the arguments are unspecified. Use `(void)` for a function declaration with no arguments.

Reply:
I will fix this. Thanks.